src/SigasiProjectCreator/ArgsAndFileParser.py

```diff
@@ -3,61 +3,223 @@
     :copyright: (c) 2008-2023 Sigasi
     :license: BSD, see LICENSE for more details.
 """
-import os
 import argparse
 import pathlib
 
+from SigasiProjectCreator import VerilogVersion, VhdlVersion
+
 
 class ArgsAndFileParser:
     options = None
 
-    def __init__(self, usage):
+    def __init__(self):
+        ArgsAndFileParser.options = None
         self.parser = argparse.ArgumentParser(prog='SigasiProjectCreator')
         self.parser.add_argument('project_name', help='Project name')
         self.parser.add_argument('input_file', help='Input file or comma-separated list of input files')
         self.parser.add_argument('destination_folder', help='Root folder of created project', type=pathlib.Path,
                                  nargs='?')
```

Comment:
It'd be better if input_file had `nargs='+'` and the destination folder was passed with e.g. `-d`. With comma-separated lists, the users do not get autocompletion; and you have to manually parse the filenames (that could contain comma's which would have to be escaped, ...)

Reply:
changed + related code to process input files and destination folder